test/Preprocessing/pp001.F

```diff
@@ -1,3 +1,4 @@
+! RUN: %flang -E %s
```

Comment:
This shouldn't be here as these tests aren't changing.

Reply:
Okay, we left them ported, but stopped them from being run. I'll remove them altogether.

Reply:
Please do not delete these test files, if that's indeed your intention.